dockstore-client/src/main/java/io/dockstore/client/cli/nested/WorkflowClient.java

```diff
@@ -267,6 +268,15 @@ protected void handlePublishUnpublish(String entryPath, String newName, boolean
     }
 
     @Override
+    protected void handleStarUnstar(String entryPath, boolean unstarRequest) {
```

Comment:
Both implementations of this seem exactly the same. 
Just put in in the abstract class if there's no difference between this and the version in ToolClient.java